core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/DetermineJoinDistributionType.java

```diff
@@ -108,7 +108,7 @@ public static double getSourceTablesSizeInBytes(PlanNode node, Context context)
     static double getSourceTablesSizeInBytes(PlanNode node, Lookup lookup, StatsProvider statsProvider, TypeProvider typeProvider)
     {
         boolean hasExpandingNodes = PlanNodeSearcher.searchFrom(node, lookup)
-                .where(isInstanceOfAny(JoinNode.class, Unnest.class))
+                .where(isInstanceOfAny(JoinNode.class, UnnestNode.class))
```

Comment:
We should make such errors impossible or at least harder to do. -> https://github.com/trinodb/trino/pull/11631